### ISSUE TITLE
[3.3] Performed Testing

### ISSUE THREAD

**Issue by KeeHanXiang** [lowyisan/INF2001_P6-6#142]
**What is the task about?** 
This task is focused on the testing of the software implemented. 

**What are the goals of the task?** 
Ensure that the system meets the requirements and required functionalities through testing methodologies. 

**Success or Completion Criteria:** 
To complete the set of tests through Black box and White box testing.  

**Allocated time:** 3 days

**Start date and end date:** [Start Date: 15/11/23], [End Date: 17/11/23]

**Owner:** Yi San & Han Xiang

**Status of the task:** To-Do

**Priority:** High

**Comment by KeeHanXiang** [comment]
Describe the work done in detail.
This task focuses on testing the software to ensure it meets requirements and functionalities through Black Box and White Box testing.

Repository

Has this work been committed to the repository?
The work does not involve any code. Therefore, no repository commits is needed.

How does the work affect success or completion criteria?
Successful testing, encompassing both Black Box and White Box methods, is essential for meeting the completion criteria, ensuring system alignment with requirements.

Start date and end date:
[Start Date: 15/11/23]
[End Date: 17/11/23]

**Comment by lowyisan** [comment]
**Describe the work done in detail.**
In my role, I concentrated on white box testing for our software, which involved a detailed examination of the logical flow of our code. My efforts were directed at ensuring the internal operations met our stringent standards for quality and performance. I created and executed test cases that covered all logical paths, including loop validations and condition checks, to verify that every aspect of the code behaved as expected.

**Has this work been committed to the repository?**
The work does not involve any code. Therefore, no repository commits is needed.

**How does the work affect success or completion criteria?**
The completion of the white box testing phase is pivotal for our success criteria. It ensures that the system is robust from a code perspective, with each function and module thoroughly vetted for accuracy. This phase's successful execution is vital to our commitment to delivering reliable and error-free software.

**Start date and end date:****
[Start Date: 11/11/23]
[End Date: 13/11/23]
